packages/maskbook/src/extension/background-script/HelperService.ts

```diff
@@ -1,5 +1,6 @@
 import { memoizePromise } from '../../utils/memoize'
 import { constructRequestPermissionURL } from '../popups'
+import { exclusiveTasks } from '../content-script/tasks'
```

Comment:
`exclusiveTasks` has been removed from the mainline but you don't need to rewrite this. I'll take care of it.

Reply:
The `exclusiveTasks` have been removed, waiting for a new API.